labguru/core.py

```diff
@@ -28,18 +28,19 @@ def __init__(self, login, password):
     """
     def add_project(self, title, description=None):
         assert isinstance(title, str) and len(title) > 0, 'title is required to create a new project'
-
         return Project(token=self.session.token, title=title, description=description).register()
 
     def get_project(self, project_id):
+        assert project_id, 'project_id is required to get project'
```

Comment:
validate_item(project_id,class)
def validate_item
assert id, 'class_id is required to get class'